.forge-snapshots/SwapRouter02ExecutorExecuteAlreadyApproved.snap

```diff
@@ -1 +1 @@
-118092
\ No newline at end of file
+109807
```

Comment:
Kinda confused why the gas is so high...Could you compare the snapshot taking out signOrder and the abi.encode stuff in the PR before? Or put it inside this snapshot. I think there will be slightly more accurate gas comparison that way

Reply:
Ah yeah the gas savings are because it's not doing signOrder and encoding :( , gas should be relatively similar then